### PR TITLE
Adding new customer qualifications and MCA direct signing scenarios

### PR DESCRIPTION
**Pull request by gaurav-karna**
# Description

Adding scenarios for `CreateCustomerQualificationWithGCC`, `CreateCustomerQualification`. `GetCustomerQualifications`, and `GetDirectSignedCustomerAgreementStatus` for March SDK release.

## Related issue

Kindly link any related issues (e.g. Fixes #xyz).

### CODE REVIEW

File: sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="CreateCustomerQualification.cs" company="Microsoft">
```
Comment (kaminasy) [inline sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs:2]: CreateCustomerQualificationWithGCC

File: sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CreateCustomerQualificationWithGCC"/> class.
+        /// </summary>
+        /// <param name="context">The scenario context.</param>
+        public CreateCustomerQualificationWithGCC(IScenarioContext context) : base("Create customer qualification with GCC", context)
+        {
+        }
+
+        /// <summary>
+        /// Executes the create customer qualification scenario.
```
Comment (kaminasy) [inline sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs:26]: Executes the create customer qualification with GCC scenario

Comment (gaurav-karna) [reply sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs]: Fixed in most [recent commit](https://github.com/microsoft/Partner-Center-DotNet-Samples/pull/70/commits/df999c86ab2e7c7602c40c01ea4c9bd2199fca33)

File: sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs (hunk 1 above)
Comment (gaurav-karna) [reply sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs]: Fixed in most [recent commit](https://github.com/microsoft/Partner-Center-DotNet-Samples/pull/70/commits/df999c86ab2e7c7602c40c01ea4c9bd2199fca33)

**Comment by gaurav-karna** [comment]
Signing off.

**Review by kaminasy — APPROVED** [review]

File: sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+            this.Context.ConsoleHelper.StartProgress("Retrieving validation codes");
+            var validations = partnerOperations.Validations.GetValidationCodes();
+            this.Context.ConsoleHelper.StopProgress();
+
+            this.Context.ConsoleHelper.Success("Success!");
+
+            this.Context.ConsoleHelper.WriteObject(validations, "Validations");
+
+            string validationCodeToRetrieve = this.ObtainQuantity("Enter validation code to use by ValidationId");
```
Comment (anush89) [inline sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs:42]: Probably better to have separate ObtainGccCode or something. I see this is reusing existing value.

File: sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                {
+                    code = c;
+                    break;
+                }
+            }
+
+            if (code == null)
+            {
+                this.Context.ConsoleHelper.Error("Code not found");
+            }
```
Comment (anush89) [inline sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs:58]: Does the code return if it is null?

**Review by anush89 — APPROVED** [review]
Approved with suggestions.

**Comment by kaminasy** [comment]
Please lmk once the PR comments are addressed.

File: sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs (hunk 4 above)
Comment (gaurav-karna) [reply sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs:58]: Yes, although the return value would be the appropriate error message. In other words, if the validation code is not instantiated, then the request still gets launched to our service. But the partner would receive a 400 if the requested qualification is GCC.

File: sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs (hunk 3 above)
Comment (gaurav-karna) [reply sdk/SdkSamples/Customers/CreateCustomerQualificationWithGCC.cs:42]: Makes sense, but this was commonly used in other files to extract user input from CMD and so I used it here.

**Pull request merged**